spec/scatter_swap_spec.rb

```diff
@@ -45,43 +45,37 @@
 end
 
 describe "#scatter" do
-  it "should return a number different from original" do
```

Comment:
This was a nonsense spec that really only passed because `original_array` was modified in place in the scatter function. The statement "should return a number different from original" was not even accurate. In some cases the scattering results in the exact same number. I have removed it and am relying on the fixtures instead.